eng/pipelines/runtime-ios-scenarios-perf-jobs.yml

```diff
@@ -138,3 +139,24 @@ jobs:
         performanceRepoAlias: ${{ parameters.performanceRepoAlias }}
         ${{ each parameter in parameters.jobParameters }}:
           ${{ parameter.key }}: ${{ parameter.value }}
+
+  # run iOS scenarios - CoreCLR
+  - template: /eng/pipelines/common/platform-matrix.yml@${{ parameters.runtimeRepoAlias }}
+    parameters:
+      jobTemplate: /eng/pipelines/templates/runtime-perf-job.yml@${{ parameters.performanceRepoAlias }}
+      buildConfig: release
+      runtimeFlavor: coreclr
+      platforms:
+        - osx_x64
+      jobParameters:
+        runtimeType: iOSCoreCLR
+        codeGenType: JIT
```

Comment:
Shouldn't this be "interpreter"?

Reply:
Thank you. I think it would be best to name the configurations same as CoreCLR names them, so using R2R instead of mono "AOT". That would bring parity to the naming we started with Android on CoreCLR.